tools/validators/instance_validator/tests/telemetry_validator_test.py

```diff
@@ -0,0 +1,102 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests tools.validators.instance_validator.telemetry_validator"""
+
+from __future__ import absolute_import
+from __future__ import print_function
+
+import os
+import json
+
+from validate import telemetry
+from validate import telemetry_validator
+from absl.testing import absltest
+
+_TELEMETRY_PATH = os.path.join('.', 'tests', 'fake_telemetry')
+
+_MESSAGE_ATTRIBUTES = os.path.join(_TELEMETRY_PATH, 'message_attributes.json')
+
+_MESSAGE_GOOD = {
+  attributes = json.loads(_MESSAGE_ATTRIBUTES),
+  data = os.path.join(_TELEMETRY_PATH, 'telemetry.json')
```

Comment:
I'm confused by this.  It appears the code under test wants an object, but this is a path?  What am I missing?

Reply:
This code wasn't intended to be reviewed yet. Tests are now done and ready for review.